guides/common/modules/proc_disabling-puppet.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Disabling_Puppet_Integration_{context}"]
+= Disabling Puppet Integration with {Project}
+
+To discontinue using Puppet in your {Project}, follow this procedure.
+
+.Prerequisites
+* You upgraded {Project} from the previous version with Puppet server enabled.
+
+.Procedure
+. Disable Puppet server on external {SmartProxies} before you disable it on {ProjectServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} plugin purge-puppet
+----
+. Disable Puppet server on {ProjectServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} plugin purge-puppet [--remove-all-data]
+----
++
+The command without arguments will remove all internal Puppet data on {ProjectServer}.
+To remove external Puppet data as well, add the `--remove-all-data` argument to the command.
```

Comment:
I'm not sure about the internal vs external data. Perhaps instead of internal we should say "Puppet related data in {Project}" and instead of external data "Puppetserver data files, including Puppet environments"